From 85f2c6956356f1c0f80f0ef6261f452711f7b2f8 Mon Sep 17 00:00:00 2001
From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 12 Feb 2018 21:30:30 +0100
Subject: ungoogled-chromium: Disable intranet detector

Disables the intranet redirect detector. It generates extra DNS requests and the functionality using this is disabled.

See this page for more information: https://mikewest.org/2012/02/chrome-connects-to-three-random-domains-at-startup

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 chrome/browser/intranet_redirect_detector.cc | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/chrome/browser/intranet_redirect_detector.cc b/chrome/browser/intranet_redirect_detector.cc
index 935d24eb60c13..0381c32366483 100644
--- a/chrome/browser/intranet_redirect_detector.cc
+++ b/chrome/browser/intranet_redirect_detector.cc
@@ -117,9 +117,7 @@ void IntranetRedirectDetector::FinishSleep() {
   simple_loaders_.clear();
   resulting_origins_.clear();
 
-  const base::CommandLine* cmd_line = base::CommandLine::ForCurrentProcess();
-  if (cmd_line->HasSwitch(switches::kDisableBackgroundNetworking))
-    return;
+  return;
 
   DCHECK(simple_loaders_.empty() && resulting_origins_.empty());
 
-- 
2.55.0

